### PR TITLE
clarify EIP 4 author

### PR DESCRIPTION
**Pull request by cdetrio**
Clarify authorship and reformat to markdown.

### CODE REVIEW

**Comment by cdetrio** [comment]
Thanks for double checking. I tracked down the git history and @ethers was the author https://github.com/ethereum/EIPs/pull/17

**Comment by cdetrio** [comment]
It is standard to change the author for this kind of derivative work (in this case, derived from BIP 123). For example [REP 1](http://www.ros.org/reps/rep-0001.html), [JEP 1](http://openjdk.java.net/jeps/1), [BEP 1](http://bittorrent.org/beps/bep_0001.html) are all derived from [PEP 1](https://www.python.org/dev/peps/pep-0001/) but have changed the author. The original author of the preexisting work should be given attribution, but the author should be the person who actually submitted the EIP.

File: EIPs/eip-4.md [hunk 1]
```diff
@@ -2,52 +2,52 @@
...
   Status: Draft
   Type: Process
   Created: 2015-11-17
 </pre>
 
-==Abstract==
+# Abstract
 
-This document describes a classification scheme for EIPs, adapted from BIP123.
+This document describes a classification scheme for EIPs, adapted from BIP123 by Eric Lombrozo.
```
Comment (axic) [inline EIPs/eip-4.md]: Doesn't this now read that BIP123 was adapted to EIP4 by Eric Lombrozo?

Comment (cdetrio) [reply EIPs/eip-4.md]: yes, will just reference with an inline link, thanks.

**Pull request merged**